wallet-new/src/Cardano/Wallet/API/Request/Filter.hs

```diff
@@ -195,6 +222,21 @@ instance ( HasServer subApi ctx
                           return $ toFilterOperations (parseQueryText $ rawQueryString req) (Proxy @params)
         in route (Proxy :: Proxy subApi) context delayed
 
+-- | Defines name of @FilterBy syms res@ as sum of @syms@,
+-- and specifies parameter 'FilterBy' provides.
+-- Used in e.g. logging.
+instance (KnownSymbols syms, syms ~ ParamNames res params) => ApiHasArgClass (FilterBy params res) where
+    type ApiArg (FilterBy params res) = FilterOperations res
+
+    apiArgName _ =
+        let filterNames = mconcat . intersperse ", " $ symbolVals (Proxy @syms)
+        in  formatToString ("filters: ("%build%")") filterNames
```

Comment:
This will log all of the names of the parameters that *could* be filtered on, not the actual filtering values. Is this what is wanted?

Reply:
I hardly think so. I believe we should log here the filters that are actually submitted with the query. I fail at seeing the value this brings otherwise.